lib/sen_tweet_web/controllers/stream_controller.ex

```diff
@@ -4,4 +4,13 @@ defmodule SenTweetWeb.StreamController do
   def index(conn, _params) do
     render(conn, "index.html", streams: Bitfeels.all_streams())
   end
+
+  def new(conn, _params) do
+    render(conn, "new.html")
+  end
+
+  def create(conn, %{"user" => user, "track" => track, "filter_level" => filter_level}) do
+    Bitfeels.twitter_stream(user, track, filter_level)
+    redirect(conn, to: "/admin/streams")
```

Comment:
Use [path helpers](https://hexdocs.pm/phoenix/routing.html#path-helpers) rather than using hardcoded strings.

This one should be `Routes.stream_path(conn, :index)`